userguide/tn_vcenter.rst

```diff
@@ -132,7 +132,8 @@ in the left tree. In the secure connection screen shown in
 Using the vCenter Plugin
 ------------------------
 
-From the main vSphere window, click
+Log in to the paired vCenter Host by typing its IP address into a
```

Comment:
Do not use possessives unless there is no other alternative.  And there is always an alternative.